src/libraries/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests.csproj

```diff
@@ -11,8 +12,6 @@
     <DefineConstants Condition="'$(TargetPlatformIdentifier)' == 'windows'">$(DefineConstants);TargetsWindows</DefineConstants>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="$(CoreLibSharedDir)System\PasteArguments.cs"
```

Comment:
These are being removed because they're actually unused?

Reply:
It seems to be some kind of leftover after refactoring.

This internal type has dedicated tests defined here:

https://github.com/dotnet/runtime/blob/018efc57d557b5234e168b5c16e6ea1153de6db2/src/libraries/Common/tests/Tests/System/PasteArgumentsTests.cs#L10

and included in:

https://github.com/dotnet/runtime/blob/018efc57d557b5234e168b5c16e6ea1153de6db2/src/libraries/Common/tests/Common.Tests.csproj#L80